### ISSUE TITLE
CoFH core not being seen

### ISSUE THREAD

**Issue by debtbringer** [CoFH/Feedback#27]
I have created my own mod pack (Mod list: pastebin.com/yFhe2fUh) and I have a couple of friends that are playing with me and most everyone got the mod pack with no issues. Except one friend whos the only one on a mac and his modpack wont even load fully here's the "Crash report": pastebin.com/Ch9SF42G I call it "Crash Report" because there is no real crash we just get the screen http://imgur.com/Zsn2xJ5

**Comment by KingLemming** [comment]
Update. Macs have problems with certain versions.

**Comment by debtbringer** [comment]
Is there a version that you can point me to that will work because ive
tried a couple of versions and they have just caused it to not even open in
a window

**Comment by Spartanyed** [comment]
I play on a mac as well I brought up the same issue.
these versions seem to be the newest and it fixed it.  


http://minecraft.curseforge.com/mc-mods/69162-cofhcore/files

**Comment by debtbringer** [comment]
Can you give me a bit a little more detail? I tried versions 99 (newest)
and it gave me the same error, and 98 and it gave me a crash, then tried 97
and it crashed again, and then tried 95 and gave me the error again.

thanks for trying to help sorry its being such a pain

**Comment by Spartanyed** [comment]
![screen shot 2015-01-28 at 11 24 40 am](https://cloud.githubusercontent.com/assets/10488362/5941650/5da827ec-a6e0-11e4-881d-b73557f33e70.png)
![screen shot 2015-01-28 at 11 25 01 am](https://cloud.githubusercontent.com/assets/10488362/5941653/5fd81374-a6e0-11e4-92b8-bf6ea3a88953.png)
![screen shot 2015-01-28 at 11 25 08 am](https://cloud.githubusercontent.com/assets/10488362/5941654/61855ce0-a6e0-11e4-9bf5-2e15feffbbc3.png)
These worked for me and sorry for not replying for so long also try search for versions on curse forge versus links on cofh site.

**Comment by talonos2** [comment]
You mention "Macs have problems with certain versions," but don't say why. If this is because you do not know why, I believe I have a solution. If you do know why, but didn't say for sake of brevity, then I apologize for being patronizing.

At any rate, I discovered a solution to let "non-mac builds" work on macs here: http://bit.ly/1BVgtKQ Basically, I believe the incompatibility has something to do with the way the jars are compressed. Unzipping the contents of the jar and rezipping them immediately with windows 7 results in a jar file with a slightly different file size that works perfectly fine on macs.

Again, if you already knew this, I apologize. Just thought I'd try to help. I spent all afternoon in this the other day.
